demo_oc3_2024/src/main.rs

```diff
@@ -34,9 +37,28 @@ pub struct Params {
     #[arg(long, value_parser = path_exists, default_value = "demo_oc3_2024/testdata/demo_evidence.binarypb")]
     pub evidence: PathBuf,
 
-    /// Path VCEK endorsing the TEE.
+    /// Path endorsements.
     #[arg(long, value_parser = path_exists, default_value = "demo_oc3_2024/testdata/demo_endorsements.binarypb")]
     pub endorsements: PathBuf,
+
+    /// Expected Sha2-256 hash of the initial measurement of the VM memory in the attestation
+    /// report.
+    #[arg(
+        long,
+        value_parser = parse_ref_value,
+        // obtained by building stage0 and running `cargo run -p snp_measurement -- --stage0-rom="./stage0_bin/target/x86_64-unknown-none/release/stage0_bin"`.
+        default_value = "f6df2054a387f3f829914196086d5992646b7cbd834270c4db205cd36879977ee06016c1e65c9ec453e334a1353e933a"
+    )]
+    pub initial_measurement: BinaryReferenceValue,
```

Comment:
However, we cannot use it as a reference value because of an issue in the verification library's implementation of reference values.

In principle it should easy to verify this reference value. The hash matches exactly the `initial_measurement` field in the attestation report contained within the extracted evidence returned by the verification library.